hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/MetricsDashboardUrlEndpoint.java

```diff
@@ -0,0 +1,64 @@
+package pl.allegro.tech.hermes.management.api;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.api.ErrorCode;
+import pl.allegro.tech.hermes.management.domain.ManagementException;
+import pl.allegro.tech.hermes.management.domain.MetricsDashboardUrl;
+import pl.allegro.tech.hermes.management.domain.MetricsDashboardUrlService;
+
+import java.util.Optional;
+
+import static jakarta.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Component
+@Path("/dashboards")
+public class MetricsDashboardUrlEndpoint {
+
+    private static final Logger logger = LoggerFactory.getLogger(MetricsDashboardUrlEndpoint.class);
+
+    private final Optional<MetricsDashboardUrlService> metricsDashboardUrlService;
+
+    MetricsDashboardUrlEndpoint(Optional<MetricsDashboardUrlService> metricsDashboardUrlService) {
+        if (metricsDashboardUrlService.isEmpty()) {
+            logger.info("Dashboards url bean is absent");
+        }
+        this.metricsDashboardUrlService = metricsDashboardUrlService;
+    }
+
+    @GET
+    @Path("/topics/{topic}")
+    @Produces(APPLICATION_JSON)
+    public Response fetchUrlForTopic(@PathParam("topic") String topic) {
```

Comment:
nit: maybe we can write small e2e tests for `fetchUrlForTopic` and `fetchUrlForSubscription`?